test/odbc/connection_object.cpp

```diff
@@ -0,0 +1,44 @@
+#include "connection_object.h"
```

Comment:
Whats the difference between this class and `ConnectionStringObject`. Could we just use one? They mostly have the same things other than `ConnectionStringObject` having the extra Test selection aprt

Reply:
My apologies, I forgot to `git add` `ConnectionStringObject` when I renamed that file. Will remove next commit.

Reply:
Done.